content/tutorials/extensions/dnn-manifest-schema/index.md

```diff
@@ -336,7 +340,7 @@ Some component types are applicable only to the package type of the same name; g
     Notes:
     * You can specify more than one globbing pattern on the same line by separating each pattern with `;`
     * More information on supported globbing patterns can be found at the [Microsoft.Extensions.FileSystemGlobbing documentation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.filesystemglobbing.matcher?view=dotnet-plat-ext-3.1#remarks)
-    * Globbing patterns can only be used to match files, not directories (if you need to delete a directory, first it needs to be empty, then you need to use one of the above methods to delete the actual directory).
+    * Globbing patterns can only be used to match files and folders inside a folder. not the folder itself.(if you need to delete a directory, first it needs to be empty, then you need to use one of the above methods to delete the actual directory).
```

Comment:
```suggestion
    * Globbing patterns can only be used to match files and folders within the extension folder (not the extension folder itself - if you need to delete a directory, first it needs to be empty, then you need to use one of the above methods to delete the actual directory).
```

Reply:
@david-poindexter actually it does not have to be in the extension folder itself. A glob could be made in a module manifest to delete skin files for instance...

Reply:
Looks good to me :-)

Reply:
> @david-poindexter actually it does not have to be in the extension folder itself. A glob could be made in a module manifest to delete skin files for instance...
Right I think so too.

Reply:
@david-poindexter I do agree the restructuring of the sentence is better :-)